### DISCUSSION TITLE
Missing `openal32.dll`

### DISCUSSION THREAD

**Discussion by TornadoTechnology** [technologists-team/hypercube#21] [General]
If you have problems with this library being unavailable, please post your OS and its bit rate here. I encountered this problem on my **64bit Windows 11**. If anything, [donwload installer](https://openal.org/downloads/)

**Comment by JerryImMouse** [comment]
I had this too

**Comment by JerryImMouse** [comment]
Provided openal32.dll with project because of failing tests in #31
